opm/simulators/flow/OutputBlackoilModule.hpp

```diff
@@ -1668,6 +1678,42 @@ class OutputBlackOilModule : public GenericOutputBlackoilModule<GetPropType<Type
                 this->fip_[Inplace::Phase::CO2MassInGasPhaseMobKrg][globalDofIdx] = 0;
             }
         }
+
+        if (!this->fip_[Inplace::Phase::CO2MassInGasPhaseMaximumTrapped].empty() || 
+            !this->fip_[Inplace::Phase::CO2MassInGasPhaseMaximumUnTrapped].empty() ) {
+            Scalar sgcr = scaledDrainageInfo.Sgcr;
```

Comment:
I suggest writing this as
```C++
auto trappedCritGas = scaledDrainageInfo.Sgcr;
```
to avoid "shadowing variable" warnings since `sgcr` [is already defined above](https://github.com/OPM/opm-simulators/blob/40ec75068df36d1e1fd10b5f0e1be8d1aea87de2/opm/simulators/flow/OutputBlackoilModule.hpp#L1604)